src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java

```diff
@@ -325,6 +325,11 @@ default String setex(byte[] key, int seconds, byte[] value) {
 
   byte[] echo(byte[] arg);
 
+  /**
+   * @deprecated This method will be removed from this interface. Use
+   * {@link AdvancedBinaryJedisCommands#move(byte[], int)}.
+   */
+  @Deprecated
   Long move(byte[] key, int dbIndex);
```

Comment:
Won't it mark the method as deprecated on BinaryJedis too?

Reply:
@gkorland No. The method will stay. It's just moved to Advanced... interface.

Reply:
I know it will stay, but BinaryJedis implement them both, which mean the IDEs might think it's deprecated no?

Reply:
@gkorland It shouldn't because there's at least one interface (Advanced...) that doesn't deprecate it.
Note: My IDE doesn't it as deprecated.